components/automate-ui/src/app/entities/infra-roles/infra-role-details.reducer.ts

```diff
@@ -39,6 +42,20 @@ export function infraRoleDetailsEntityReducer(
     case RoleActionTypes.GET_FAILURE:
       return set(GET_STATUS, EntityStatus.loadingFailure, state);
 
```

Comment:
[ nit ] - not your fault, but while you're in here can you fix the indentation from L39-L44 so that it matches the rest of the indents in the switch case

Reply:
updated the changes.